internal/cmd/config.go

```diff
@@ -1043,9 +1055,9 @@ func (c *Config) defaultPreApplyFunc(
 
 // defaultSourceDir returns the default source directory according to the XDG
 // Base Directory Specification.
-func (c *Config) defaultSourceDir(fileSystem vfs.Stater, bds *xdg.BaseDirectorySpecification) (chezmoi.AbsPath, error) {
+func (c *Config) defaultSourceDir(fileSystem vfs.Stater) (chezmoi.AbsPath, error) {
 	// Check for XDG Base Directory Specification data directories first.
-	for _, dataDir := range bds.DataDirs {
+	for _, dataDir := range c.xdgDataDirs {
```

Comment:
I would be very cautious about using `xdgDataDirs`, as it points to directories outside of `$HOME` and does not clearly include `$XDG_DATA_HOME` in the documentation for adrg/xdr. Chezmoi is a *personal* dotfiles manager and should not be accessing anything outside of `$HOME` or *maybe* `/tmp` (`/private/tmp` on macOS).

Reply:
Indeed, `xdg.DataDirs` does not include `xdg.DataHome`, but that is why i'm using the `xdgDataDirs` field to also include `xdg.DataHome` and retain the behaviour that was present in `go-xdg`.